src/App.tsx

```diff
@@ -1,56 +1,91 @@
-import React from 'react';
-import './App.scss';
-import { peopleFromServer } from './data/people';
+/* eslint-disable @typescript-eslint/quotes */
```

Comment:
just use single quotes instead of disabling lint rule

Reply:
I'd love to, but for some reason my eslint automatically formats in double quotes. I haven't found a way to fix it...